frontend/src/components/Notificacao/Consulta/NotificacaoTable/index.vue

```diff
@@ -189,15 +191,33 @@ export default {
     isPermiteEvoluir(item) {
       return item.status === 'ABERTA' && this.isSecretariaSaude;
     },
+    isPermiteVisualizar(item) {
+      return this.isUnidadeSaudePermitidaUserLogged(item.unidadeSaudeId) || this.isSecretariaSaude;
+    },
+    isPermiteExcluir() {
+      return this.isSecretariaSaude;
+    },
     updateFiltroSituacao(status) {
       this.filtroStatus = status;
       this.options = { ...this.options, page: 1 };
       this.consultarNotificacoes();
     },
+    isUnidadeSaudePermitidaUserLogged(unidadeSaudeId) {
+      return this.unidadesSaudeUserLogged.some((data) => data.id === unidadeSaudeId);
+    },
+    consultarUnidadesSaudeUsuario() {
+      UnidadeSaudeService.findByUserEmail(keycloak.tokenParsed.email)
+        .then(({ data }) => {
+          this.unidadesSaudeUserLogged = data;
```

Comment:
Se esse filtro for restritivo (somente ver as notificações da unidade de saúde do usuário), deve ser feito no back principalmente. Senão, um pentest simples já que mata.